src/main/src/main/java/org/geoserver/logging/LoggingUtils.java

```diff
@@ -71,34 +67,42 @@ public static void configureGeoServerLogging(
 
         Properties lprops = new Properties();
         lprops.load(loggingConfigStream);
+
+        // look up log file location
+        if (logFileName == null) {
+            logFileName = loader.get("logs").get("geoserver.log").file().getAbsolutePath();
+        } else {
+            if (!new File(logFileName).isAbsolute()) {
+                logFileName = new File(loader.getBaseDirectory(), logFileName).getAbsolutePath();
+                LoggingInitializer.LOGGER.fine(
+                        "Non-absolute pathname detected for logfile.  Setting logfile relative to data dir.");
```

Comment:
@aaime is this is the log message you suspect we should disable?

Reply:
I don't have suspicions about it... came to this PR at the end of my allotted time and only had a glance. It just weirded me out that the new code would nag about log file location even when file logging is disabled.

Reply:
Okay I will see what I can do, I should be more mean with “fine” messages, nobody is really checking for those in the output.

Reply:
And thanks for the review, I was so focused on preserving the functionality, I did not look at the bigger picture of if the log messages were you know … useful.